tests/unit/test_sample_erratum.py

```diff
@@ -689,6 +692,38 @@ def test_can_load_erratum(data_path):
         references=[],
         pkglist=[],
         content_type_id="erratum",
+        container_list=frozenlist(
```

Comment:
You shouldn't have to explicitly put the frozenlist/frozendict here and it'd probably be better if you didn't - there's supposed to be an automatic conversion anyway when the object is initialized, and I think that's how it would generally be used.  The usage of frozenlist/frozendict is somewhat hidden (caller doesn't care about it, as long as they never try to mutate the data).